packages/panzer/disc-fe/src/Panzer_IntegrationValues2.cpp

```diff
@@ -1121,6 +1121,7 @@ getCubatureCV(const PHX::MDField<Scalar,Cell,NODE,Dim>& in_node_coordinates,
   }
 
   size_type num_cells = in_num_cells < 0 ? in_node_coordinates.extent(0) : (size_type) in_num_cells;
+  std::pair<int,int> cell_range(0,num_cells);
```

Comment:
Wouldn't it be better to use Kokkos::pair instead?

Reply:
> Wouldn't it be better to use Kokkos::pair instead?

Not according to the kokkos wiki subview page:

In the above example and those that follow in this chapter, we assume that Kokkos::View, Kokkos::subview, Kokkos::ALL, std::make_pair, and std::pair have been imported into the working C++ namespace.

Reply:
I should comment that all subviews are being created on host, outside of kokkos kernels. Maybe you need the kokkos version of pair inside device kernels?

Reply:
@rppawlo I see, that makes sense, thanks. I guess on device it would be safer to use Kokkos::pair.